core/views.py

```diff
@@ -91,42 +91,43 @@ def get_context_data(self, **kwargs):
                     ).annotate(
                     sum_goals=Sum(
                         Case(
-                            When(stat__team=F('roster__team'), then=Coalesce('stat__goals',0)),
+                            When(stat__team=F('roster__team'), stat__team__season__is_current_season=True, then=Coalesce('stat__goals',0)),
```

Comment:
I think we actually want to filter on team__is_active not season__is_current_season

It's probably bad db design, but the way it works is, there is a new team_id created for each season. It has a season and division id. The issue is the draft league and the sunday league, don't always end at the same time. So right now we're on summer season for Sunday, but still on the spring season for Wednesday draft.